projects/kit/src/lib/time/types/time-segments.ts

```diff
@@ -0,0 +1,6 @@
+export interface MaskitoTimeSegments<T = string> {
```

Comment:
What's the point of generic here? Shouldn't it just be `string | number`?

Reply:
This type is used many times in the code.
Sometimes it is convenient to have object with all strings. Sometimes with all numbers.
And don't do extra type checks.